packages/cdk-assets/lib/aws-operations.ts

```diff
@@ -0,0 +1,18 @@
+import * as AWS from 'aws-sdk';
+
+/**
+ * AWS SDK operations required by Asset Publishing
+ */
+export interface IAws {
+  defaultRegion(): Promise<string>;
+  currentAccount(): Promise<string>;
+
+  s3Client(options: ClientOptions): AWS.S3;
```

Comment:
Didn't we say we wanted to avoid exposing SDK types in the public API (for jsii's sake)?